tracee-ebpf/main.go

```diff
@@ -175,16 +175,24 @@ Use this flag multiple times to choose multiple capture options
 		outputParts := strings.SplitN(o, ":", 2)
 		numParts := len(outputParts)
 		if numParts == 1 {
+			if !validateOutputFormatValue(outputParts[0]) {
```

Comment:
Instead of validating when we get inside `prepareOutput` it will be easier to validate outside, prior to reaching here. So I suggest that before we call `prepareOutput` we can pass `c.StringSlice(output)` to another function that can validate all types of possible inputs that can be passed to `--output` flag.

This will keep the abstraction of `prepareOutput` to assume the inputs it receives are valid and also make it easier to read and test as it will have less branches. Furthermore testing will be simplified as well as we can just test both functions individually.

Reply:
regarding validation, I don't think that preparer functions should have validation logic. the XXXConfig struct already has the validation logic https://github.com/aquasecurity/tracee/blob/f8df7da6a27f729610992b6bd52e89d510fcf384/tracee-ebpf/tracee/tracee.go#L114 so we should just use it whenever is needed. 

also I think other configs should follow this pattern. wdyt @simar7 ?

Reply:
Thanks @simar7 . I used the Validate() method and removed the other method that I created.